### DISCUSSION TITLE
Really discover anonymous Blade components in other folders

### DISCUSSION THREAD

**Discussion by misog** [laravel/framework#42819] [General]
Hi, how can I set up auto-discovery of anonymous Blade components in custom folders?

I know that [this pull request](https://github.com/laravel/framework/pull/41637) has `Blade::anonymousComponentNamespace` however with that I could not discover and use anonymous components from other folders **with empty namespace**.

I want to use a component in `app/custom/path/image.blade.php` like this:
```
<x-image></x-image>
```

Besides custom paths, this is also important for modular Laravel applications with one ServiceProvider per Module. Components in a module should not contain prefix just because they are in a module. Because when module/folder is renamed, or components are moved/copied to another module, all component references must be edited.

So it would be nice to have a method to set custom paths for auto-discovery. This applies to class components too.

As a workaround, folder symlink could be created in `resources/views/components` if the current auto-discovery supports symlinks.

**Comment by Rusbelito** [comment]
don't work , right ?
